spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/sql/GcpCloudSqlAutoConfiguration.java

```diff
@@ -207,9 +207,8 @@ public DataSourceProperties cloudSqlDataSourceProperties(
 				setCredentialsFileProperty(gcpCloudSqlProperties, gcpProperties);
 			}
 
-			System.setProperty(SslSocketFactory.USER_TOKEN_PROPERTY_NAME,
-					"spring-cloud-gcp-sql/"
-							+ this.getClass().getPackage().getImplementationVersion());
+			CoreSocketFactory.setApplicationName("spring-cloud-gcp-sql/"
```

Comment:
While you are here, would it make sense to turn it into a constant, so it's easy to see at a glance on top of the class?